src/endpoint/s3/ops/s3_put_object_tagging.js

```diff
@@ -11,7 +11,8 @@ async function put_object_tagging(req, res) {
     const reply = await req.object_sdk.put_object_tagging({
         bucket: req.params.bucket,
         key: req.params.key,
-        tagging: tag_set
+        tagging: tag_set,
+        version_id: req.query.versionId,
```

Comment:
Actually, I think I missed that one, I think that putting is not supported in was, and we should also not support it.
can you revert the version id handle in put object tagging? 
one thing to notice, is that namespace_s3 does add this into it's params, we should consider removing it from there (as a different task in a different PR).

Reply:
ok, Will remove version related changes from `put_object_tagging()`